arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dtsi

```diff
@@ -509,18 +509,18 @@
 
         camera2: camera-module@36 {
                 status = "okay";
-                compatible = "ovti,ov5647";
-                reg = <0x36>;
+                compatible = "arducam,arducam-pivariety";
+                reg = <0x0c>;
 
                 clocks = <&cru SCLK_CIF_OUT>;
-                clock-names = "clk_cif_out";
-                //pinctrl-names = "rockchip,camera_default";
+                clock-names = "xclk";
+			          clock-frequency = <24000000>;
```

Comment:
Can you be consistent with your indentation?

Reply:
Done